### PR TITLE
(GH-1) Add build script

### LINKED ISSUE #1: Add build script

### PR DESCRIPTION
**Pull request by pascalberger**
Add build script and README

Failing build is expected as solution will follow in a subsequent PR.

Fixes #1

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+```bash
+./build.sh
+```
+
+## Release
+
+See [Cake.Recipe documentation] how to create a new release of this addin.
+
+[GitFlow]: (http://nvie.com/posts/a-successful-git-branching-model/)
```
Comment (x-jokay) [inline CONTRIBUTING.md]: Use **https** for the link and **(** **)** can be removed.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+
+[![Coverage Status](https://coveralls.io/repos/github/cake-contrib/Cake.Markdownlint/badge.svg?branch=develop)](https://coveralls.io/github/cake-contrib/Cake.Markdownlint?branch=develop)
+
+## Quick Links
+
+- [Documentation](https://cake-contrib.github.io/Cake.Markdownlint)
+
+## Chat Room
+
+Come join in the conversation about this addin in our Gitter Chat Room
```
Comment (x-jokay) [inline README.md]: Add period.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,39 @@
-# Cake.Markdownlint
-Cake Addin for calling Markdownlint
+# Markdownlint Addin for Cake
+
+This addin for the Cake Build Automation System allows you to run [markdownlint] using [markdownlint-cli]
```
Comment (x-jokay) [inline README.md]: Add period.

File: build.ps1 [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+Performs a dry run of the build script.
+No tasks will be executed.
+.PARAMETER Mono
+Tells Cake to use the Mono scripting engine.
+.PARAMETER SkipToolPackageRestore
+Skips restoring of packages.
+.PARAMETER ScriptArgs
+Remaining arguments are added here.
+.LINK
+http://cakebuild.net
```
Comment (x-jokay) [inline build.ps1]: Use **https** link.

**Comment by pascalberger** [comment]
@x-jokay Ready for another review

**Pull request merged**